### PR TITLE
AAP-2423: Add repo info for installing ansible-builder

### PR DESCRIPTION
**Pull request by lmalivert**
Jira: [AAP-2423](https://issues.redhat.com/browse/AAP-2423)

- Added prerequisites section with link to ansible-builder repo and subscription info.
- Added note about encountering issues with PyPI install.

Preview link: [2.1 Installing Ansilbe Builder](http://file.rdu.redhat.com/~lmodemal/aap-builder-221304/master.html)

- Needs verification from SME
- Needs peer review

### CODE REVIEW

File: downstream/modules/builder/proc-installing-builder.adoc [hunk 1]
```diff
@@ -2,12 +2,22 @@
...
+
+.Procedure
+In your terminal, run the following command:
 ----
 $ dnf install ansible-builder
 ----
 
 You can also install {Builder} from the https://pypi.org/project/ansible-builder/[Python Package Index (PyPI)]. To install with this setup, run:
+
+NOTE: If you encounter issues while installing from PyPI, please install and reproduce on a Supported installation.
```
Comment (ariordan-redhat) [inline downstream/modules/builder/proc-installing-builder.adoc:19]: Should the note follow the `pip install` command?

**Review by ariordan-redhat — COMMENTED** [review]
Just one question. 
Otherwise, looks good to me.

File: downstream/modules/builder/proc-installing-builder.adoc (hunk 1 above)
Comment (lmalivert) [reply downstream/modules/builder/proc-installing-builder.adoc]: You're right @ariordan-redhat. I fixed it.

File: downstream/modules/builder/proc-installing-builder.adoc [hunk 2]
```diff
@@ -2,12 +2,22 @@
...
 $ dnf install ansible-builder
 ----
 
 You can also install {Builder} from the https://pypi.org/project/ansible-builder/[Python Package Index (PyPI)]. To install with this setup, run:
+
 ----
 $ pip install ansible-builder
 ----
+
+NOTE: If you encounter issues while installing from PyPI, please install and reproduce on a Supported installation.
```
Comment (ariordan-redhat) [inline downstream/modules/builder/proc-installing-builder.adoc:23]:
```suggestion
NOTE: If you encounter issues while installing from PyPI, please install and reproduce on a supported installation.
```

Should the S in Supported be uppercase?

**Review by msilmser — APPROVED** [review]
looks good to me!

**Comment by lmalivert** [comment]
Hi @msilmser! Thanks for reviewing. I noticed some missing information in my current PR and just revised. Would you mind taking a look?

**Review by TarikFD — APPROVED** [review]
LGTM!

**Pull request merged**